bc_obps/reporting/api/validate_user_report_version.py

```diff
@@ -0,0 +1,36 @@
+from enum import Enum
+from typing import Optional
+
+from ninja import Query
+from django.http import HttpRequest
+from registration.api._user_operators._current.has_registered_operation import (
+    get_current_user_operator_has_registered_operation,
+)
+from .router import router
+from ..models import ReportVersion
+from ..schema.validate_reporting_user import UserStatusResponse
+
+
+class UserStatusEnum(Enum):
+    REGISTERED = "Registered"
+    NOT_REGISTERED = "Not Registered"
+    REGISTERED_AND_VALID = "Registered and Valid Report Version"
+    REGISTERED_AND_INVALID = "Registered and Invalid Report Version"
+
+
+@router.get("/validate_user", response=UserStatusResponse)
```

Comment:
this is much too generic, I'd do: `/report-version/{version_id}/user_operation_status`

Reply:
this way no need to pass query params

Reply:
API route should match file name for this one